src/raiden_libs/cli.py

```diff
@@ -187,7 +188,9 @@ def blockchain_options(contracts: List[str]) -> Callable:
         CONTRACT_MONITORING_SERVICE: "monitor-contract",
         CONTRACT_ONE_TO_N: "one-to-n-contract",
         CONTRACT_SERVICE_REGISTRY: "service-registry-contract",
+        CONTRACT_DEPOSIT: "service-deposit-contract",
```

Comment:
what about `service-registry-deposit-contract`?

Reply:
I thought "user-deposit" and "service-deposit" were a nice pair, and it's a bit shorter. But I don't really mind either way.

Reply:
Fine for me as well. BTW, how do people get the address? Do they have to filtere the events themselves?

Can we log it during registration at least?

Reply:
I used Etherscan when testing. I wanted to get this quickly done, so I skipped event filtering and automated tests. Both make sense in later PRs.